docs/recipes/github-actions.md

```diff
@@ -40,51 +38,45 @@ jobs:
         run: npx semantic-release
 ```
 
-Anoter option is to use a trigger [`repository_dispatch`](https://help.github.com/en/articles/events-that-trigger-workflows#external-events-repository_dispatch) to have control when you want to generate a release only making a HTTP request, e.g.:
-
-```yaml
-name: Release
-on: [repository_dispatch]
-jobs:
-# ...
-```
-
-So you just call (with your personal `GH_TOKEN` or from a generic/ci user):
-
-```
-$ curl -v -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${GH_TOKEN}" https://api.github.com/repos/[org-name-or-username]/[repository]/dispatches -d '{ "event_type": "any" }'
-```
-
-And `Release` workflow will be triggered.
-
 ### `package.json` configuration
 
 A `package.json` is required only for [local](../usage/installation.md#local-installation) **semantic-release** installation.
 
-Package [`@semantic-release/git`](https://github.com/semantic-release/git) is optional but **recommended** to push your `package.json` version changes to master before generate GitHub release.
-
 ```json
 {
    "release":{
       "plugins":[
          "@semantic-release/commit-analyzer",
          "@semantic-release/github",
          "@semantic-release/npm",
-         "@semantic-release/release-notes-generator",
-         [
-            "@semantic-release/git",
-            {
-               "assets":[
-                  "package.json"
-               ],
-               "message":"chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
-            }
-         ]
+         "@semantic-release/release-notes-generator"
       ]
    },
```

Comment:
You can remove the `"release"` settings altogether, as these are now the default settings. In fact, you can remove the section on `package.json` altogether, all you need is to install `semantic-release` as devDependency

Reply:
Indeed.. makes sense! I'll change it!